core/src/main/java/org/elasticsearch/search/aggregations/AggregatorBuilder.java

```diff
@@ -95,7 +95,7 @@ public final AB readFrom(StreamInput in) throws IOException {
         // NORELEASE remove when all aggregations have StreamInput constructor
```

Comment:
Do we actually still need this method? I thought all the aggregations had now been refactored to have a StreamInput constructor?